docs/getting_started/working_in_github.md

```diff
@@ -0,0 +1,48 @@
+# Getting the code
+
+-   Create your own Fork (once) (in web ui)
+-   Clone your fork locally
+    -   git clone https://github.com/\<your_id>/connectedhomeip.git
+-   Add the Upstream as a Remote
+    -   git remote add upstream https://github.com/project-chip/connectedhomeip
+
+# Updating
+
+-   git pull upstream / git fetch upstream
+-   We use submodules with submodules
+    -   git submodule update -f --init --recursive
+-   May need to occasionally sync
+    -   git submodule sync --recursive
+
+# Pull Requests and CI
+
+## Pull Request process
+
+-   make changes locally, push to a branch on your fork
```

Comment:
Again, we are a bit inconsistent with upper/lower case staring here.

Reply:
File removed,.